tests/e2e/e2e_test.go

```diff
@@ -137,16 +126,44 @@ func (s *IntegrationTestSuite) Test03SuperfluidVoting() {
 	)
 }
 
+// TestAddToExistingLockPostUpgrade tests lockups to both regular and superfluid locks.
+// Specifically, we ensure addToExistingLock works for both preupgrade and postupgrade locks.
+func (s *IntegrationTestSuite) TestAddToExistingLockPostUpgrade() {
+	chainA := s.configurer.GetChainConfig(0)
+	chainANode, err := chainA.GetDefaultNode()
+	s.NoError(err)
+	if !s.skipUpgrade {
+		// ensure we can add to existing locks and superfluid locks that existed pre upgrade on chainA
+		// we use the hardcoded gamm/pool/1 as this was the one that was created pre upgrade
+		lockupWalletAddrA, lockupWalletSuperfluidAddrA := chainANode.GetWallet("lockup-wallet"), chainANode.GetWallet("lockup-wallet-superfluid")
+		chainANode.AddToExistingLock(sdk.NewInt(1000000000000000000), "gamm/pool/1", "240s", lockupWalletAddrA)
+		chainANode.AddToExistingLock(sdk.NewInt(1000000000000000000), "gamm/pool/1", "240s", lockupWalletSuperfluidAddrA)
+	}
```

Comment:
Do you think we could move all this pre-upgrade dependent logic into its own test  fixture `TestAddToExistingLockPreUpgrade` while renaming this to `TestAddToExistingLock`?

The motivation is twofold:
- completely decouple upgrade dependent and independent tests
- if we were to run `make test-e2e-short` with the current design, the name of `TestAddToExistingLockPostUpgrade` would not make sense because the upgrade tests are not run at all

Reply:
Good idea! Makes sense to split these two apart even further, addressing now!